generator/src/routes/+page.svelte

```diff
@@ -1,59 +1,230 @@
+  import Toggle from "$components/Toggle/index.svelte"
+  import logo from "$assets/logo.svg"
+  import effect from "$assets/effect.svg"
+  import asas from "$assets/asas.svg"
+  import logoNoTag from "$assets/logo_no_tag.png"
+  import subtract from "$assets/subtract.png"
+  import type { PageData } from "./$types"
 
-<svelte:head>
-  <title>Home</title>
-  <meta name="description" content="Svelte demo app" />
-</svelte:head>
+  export let data: PageData
 
-<section>
-  <h1>
-    <span class="welcome">
-      <picture>
-        <source srcset={welcome} type="image/webp" />
-        <img src={welcome_fallback} alt="Welcome" />
-      </picture>
-    </span>
+  let productOption: Option
+  const onToggleProduct = (option: Option) => {
+    productOption = option
+  }
 
-    to your new<br />SvelteKit app
-  </h1>
+  let signOption: Option
+  const onToggleSign = (option: Option) => {
+    signOption = option
+  }
+</script>
 
-  <h2>
-    try editing <strong>src/routes/+page.svelte</strong>
-  </h2>
+<svelte:head>
+  <title>Generator</title>
+  <meta name="description" content="QRcode Generator" />
+</svelte:head>
 
-  <Counter />
-</section>
+<div class="container">
+  <!-- Left Panel -->
+  <div class="form-wrapper">
+    <div class="title">Generate a product passport</div>
+    <img class="asas" src={asas} alt="" />
+    <form>
+      <div class="row">
+        <FormInputGroup name="domain" label="Issuer domain" placeholder="ex.tags.ioxio.dev" />
+      </div>
+      <div class="row">
+        <div class="toggle-row">
+          <Toggle options={data.productOptions} onToggle={onToggleProduct} value={productOption} />
+        </div>
+        <FormSelectGroup
+          name="product"
+          label="Product"
+          placeholder="Type a product"
+          options={data.options}
+        />
+      </div>
+      <div class="row">
+        <FormInputGroup name="productId" label="Product ID" placeholder="ex. VV123456-12" />
+        <div class="toggle-row">
+          <Toggle options={data.signOptions} onToggle={onToggleSign} value={signOption} />
+        </div>
+      </div>
+      <div class="row">
+        <FormCheckbox name="valid" label="Create valid signature" />
+      </div>
+      <div class="actions-wrapper">
+        <Button title="Generate IOXIO Tag" onClick={() => {}} />
+      </div>
+    </form>
+  </div>
+  <!-- Right Panel -->
+  <div class="qrcode-area-wrapper">
+    <div class="qrcode-area">
+      <div class="qrcode">
+        <img class="frame" src={subtract} alt="" />
+        <img class="effect" src={effect} alt="" />
+        <img class="logo" src={logo} alt="" />
+      </div>
+    </div>
+    <div class="status">IOXIO Tag generator</div>
+    <div class="product-area">
+      <div class="description">
+        <div>
+          This application serves as a demonstration platform for generating QR codes specifically
+          designed for Data Product Passports.<br />Utilizing the IOXIO Dataspace technology, this
```

Comment:
I think the design has a flaw here, we should not use `<br/>` but `</p><p>/`, I've asked for this to be fixed in the design as well but `<br/>` always looks bad in text.